data/assets/scene/solarsystem/missions/jwst/actions.asset

```diff
@@ -0,0 +1,157 @@
+    for _,v in pairs(list) do 
+      openspace.setPropertyValueSingle(v, not openspace.getPropertyValue(v)) 
+    end
+  ]],
+  Documentation = "Toggle James Webb Space Telecope field of view and view band",
+  GuiPath = "/JWST",
+  isLocal = false
+}
+
+local setup_launch = {
+  Identifier = "os.missoins.jwst.setup.launch",
+  Name = "Set to JWST launch time",
+  Command = [[
+    openspace.time.setDeltaTime(1); 
+    openspace.time.setTime('2021-12-25T12:20:01');
+  ]],
+  Documentation = "Set the time to the launch time of JWST",
+  GuiPath = "/JWST",
+  isLocal = false
+}
+
+local setup_detach = {
+  Identifier = "os.missions.jwst.setup.detach",
+  Name = "Set to JWST detach time",
+  Command = [[
+    openspace.time.setDeltaTime(1); 
+    openspace.time.setTime('2021-12-25T12:50:00');
+  ]],
+  Documentation = "Set the time to the detach time of JWST",
+  GuiPath = "/JWST",
+  isLocal = false
+}
+
+local toggle_sun_trail = {
+  Identifier = "os.missions.jwst.togglesuntrail",
+  Name = "Toggle JWST Sun trail",
+  Command = [[
+    local value = openspace.getPropertyValue('Scene.JWSTSunTrail.Renderable.Enabled'); 
+    openspace.setPropertyValueSingle('Scene.JWSTSunTrail.Renderable.Enabled', not value);
+  ]],
+  Documentation = "Toggle JWST trail relative to the Sun",
+  GuiPath = "/JWST",
+  isLocal = false
+}
+
+local toggle_trails_except_moon = {
+  Identifier = "os.missions.jwst.toggletrialsexceptmoon",
+  Name = "Toggle trails (except Moon)",
+  Command = [[
+    local list = openspace.getProperty('{planetTrail_solarSystem}.Renderable.Enabled'); 
+    for _,v in pairs(list) do 
+      openspace.setPropertyValueSingle(v, not openspace.getPropertyValue(v)) 
+    end 
+    local moonlist = openspace.getProperty('{moonTrail_solarSystem}.Renderable.Enabled') 
+    for _,v in pairs(moonlist) do 
+      openspace.setPropertyValueSingle(v, not openspace.getPropertyValue(v)) 
+    end 
+    openspace.setPropertyValueSingle('Scene.MoonTrail.Renderable.Enabled', true)
+  ]],
+  Documentation = "Toggle all planet and moon trails, except the Moon",
+  GuiPath = "/JWST",
+  isLocal = false
+}
+
+local toggle_jwst_trail = {
+  Identifier = "os.missions.jwst.togglejwsttrails",
+  Name = "Toggle JWST trail",
+  Command = [[
+    local list = {'Scene.JWSTTrailLaunch.Renderable.Enabled', 'Scene.JWSTTrailCruise.Renderable.Enabled', 'Scene.JWSTTrailCoRevOrbit.Renderable.Enabled'}; 
+    for _,v in pairs(list) do 
+      openspace.setPropertyValueSingle(v, not openspace.getPropertyValue(v)); 
+    end
+  ]],
+  Documentation = "Toggle JWST launch, cruise and L2 co-revolving orbit trails, not the Sun trail",
+  GuiPath = "/JWST",
+  isLocal = false
+}
+
+local actionlist = { 
+  toggle_lagrangian_points, 
+  toggle_hudf, 
+  toggle_l2, 
+  toggle_fov, 
+  setup_launch, 
+  setup_detach, 
+  toggle_sun_trail, 
+  toggle_trails_except_moon, 
+  toggle_jwst_trail  
+}
+
+asset.onInitialize(function()
+  for _, action in ipairs(actionlist) do
```

Comment:
I'd suggest to rather have a lot of `registerAction` calls instead;  just in case a user comes across this file and wants to use it as an example;  even though it is more verbose it would be easier to  understand